src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts
+category: Assistive technology
```

Comment:
```suggestion
category: Background
```

Let's update the category, since the article suggests Dyslexic fonts may not be effective assistive tech.